ard_controller/ard_controller.ino

```diff
@@ -278,21 +388,57 @@ void sensor_data_response() {
 }
 
 void setup() {
-  Serial.begin(115200);
-  Serial1.begin(115200);
+  Serial.begin(115200); // console
+  Serial1.begin(115200); // raspi comms
+
+  // start serial connection to ODrives
+  odrv_leftleg_ser.begin(115200);
+  odrv_rightleg_ser.begin(115200);
+  odrv_leftarm_ser.begin(115200);
+  odrv_rightarm_ser.begin(115200);
+
+  // set all ODrive axes to closed loop control
+  int requested_state = ODriveArduino::AXIS_STATE_CLOSED_LOOP_CONTROL;
```

Comment:
The more I am thinking about this, we should probably do the index search using Arduino as well and not on Odrive Bootup. This way everything is synchronized in one place. Thoughts?

Reply:
I'd say for now leave it on ODrive bootup. It wouldn't be as simple as just adding it to the `setup` function in Arduino since then we have to rely on the race condition of the arduino vs the odrive booting up first for it to be successful. As it stands, these `CLOSED_LOOP_CONTROL` calls don't set the state most of the time since the arduino usually boots first. What needs to happen is that these setup commands get moved to the main loop in such a manner that they are only run if someone enters a command in the user-facing serial console (like how it's setup in the testing scripts).

Reply:
I was thinking more along the lines that Raspi should tell Arduino to initialize the Odrive. But I see what you are saying. It makes sense

Reply:
That would probably be the best way to go in terms of ease of functionality but also the most work to pull off